App/SB/views/UserProfile/index.js

```diff
@@ -156,7 +164,8 @@ const mapStateToProps = (state, ownProps) => {
 
 const mapDispatchToProps = (dispatch) => {
   return {
-    lockScreen: () => { dispatch(AuthActions.logOutRequest()) }
+    lockScreen: () => { dispatch(AuthActions.logOutRequest()) },
+    changeAvatar: () => { dispatch(PreferencesActions.changeAvatarRequest()) }
```

Comment:
Don't see where this is called. Do we need `PreferencesActions.changeAvatarRequest` to exist?